dotCMS/src/main/java/com/dotmarketing/db/HibernateUtil.java

```diff
@@ -889,7 +889,46 @@ public static void startTransaction()  throws DotHibernateException{
 		}
 	}
 
-	public static boolean commitTransaction()  throws DotHibernateException{
+	/**
+	 * This just commit the transaction and process the listeners.
+	 * @throws DotHibernateException
+	 */
+	public static void commitTransaction()  throws DotHibernateException{
+
+		try{
+			// if there is nothing to close
+			if (sessionHolder.get() == null){
+				return;
+			}
+			Session session = getSession();
+
+			if (session != null) {

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this expression which always evaluates to "true" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2589)